parser/src/main/scala/aqua/parser/lexer/ValueToken.scala

```diff
@@ -208,25 +297,27 @@ object ValueToken {
   val literal: P[LiteralToken[Span.S]] =
     P.oneOf(bool.backtrack :: float.backtrack :: num.backtrack :: string :: Nil)
 
+  def brackets(basic: P[ValueToken[Span.S]]): P[ValueToken[Span.S]] =
+    basic.between(`(`, `)`).backtrack
+
+  val atom: P[ValueToken[S]] = P.oneOf(
+    literal.backtrack ::
+      initPeerId.backtrack ::
+      P.defer(
+        CollectionToken.collection
+      ) ::
+      P.defer(CallArrowToken.callArrow).backtrack ::
+      P.defer(brackets(InfixToken.mathExpr)) ::
+      varLambda ::
+      Nil
+  )
+
+  // One of entry points for parsing the whole math expression
```

Comment:
Comment for `_value` is the same as for `value`, but they're different. Are both used as entry points? Why have two?
And things like `foo()` or `"hello"` have nothing about math.